### ISSUE TITLE
Risk in the api

### ISSUE THREAD

**Issue by mmlngl** [ContactTracing-app/Graphql-api#9]
We want to return the risk lever for a User. This requires a new `Query` in graphql schema. From Cypher, we need to return a string matching a new enum type.

`SAFE` - all of my contacts are `healthStatus.TOTALLY_FINE`
`LOW` - any of my Indirect contacts `healthStatus.IN_QUARATINE`
`MEDIUM` - any of my contacts `healthStatus.IN_QUARATINE`
`HIGH` - Direct contacts `healthStatus.TESTED_POSITIVE`

exact strings may vary so check the schema file.

**Comment by lisabang** [comment]
@mmmoli makes sense to me.  feel free to assign to me and whoever else you think might be able to help

**Comment by mmlngl** [comment]
@kavyvetri Are you happy to pick this up with @lisabang ?

**Comment by mmlngl** [comment]
I'm gonna remove @lisabang while we regroup…

**Comment by mmlngl** [comment]
@all-contributors please add @lisabang and @kavyvetri to code

**Comment by mmlngl** [comment]
@all-contributors please add @kavyvetri for code
